js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -1573,6 +1573,9 @@
     resetDisplaySettings: function () {
       var graphName = $('#editGraphName').val();
 
+      const localStorageKey = graphName + "-gv-urlparameters";
```

Comment:
It would be better to define a central method / location for that `"-gv-urlparameters";` string. 
As any local file change will break all other occurrences. While this works and is technically correct,
it makes the code maintenance really hard.